books/RayTracingInOneWeekend.html

```diff
@@ -2884,6 +2936,23 @@
 In service of this, we'll create a new vector method -- `vec3::near_zero()` -- that returns true if
 the vector is very close to zero in all dimensions.
 
+We'll need to use the C++ standard library function `std::fabs`, which returns the absolute value of
+its input. We'll add this to `rtweekend.h` since we'll use this in several future locations.
+
+
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
+    // C++ Std Usings
+
+
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
+    using std::fabs;
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
+    using std::make_shared;
+    using std::shared_ptr;
+    using std::sqrt;
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    [Listing [declare-fabs]: <kbd>rtweekend.h</kbd> Declaring std::fabs()
```

Comment:
Missing closing `]` 

<img width="734" alt="image" src="https://github.com/RayTracing/raytracing.github.io/assets/9646385/3afd66ea-faaa-45d1-ac7f-ffb67c810246">

Reply:
Fixed in second commit.

Reply:
Oops — no it's not.